bin/check-sqs-messages.rb

```diff
@@ -49,6 +49,12 @@ class SQSMsgs < Sensu::Plugin::Check::CLI
          description: 'A comma seperated list of the SQS queue(s) you want to check the number of messages for',
          default: ''
 
+  option :exclude_queue,
```

Comment:
I actually did not catch it but it's better to name them plural where they are supporting multiples. As I have not released this yet I am gonna change those up as well prior to release.